workflows/test_flows/main.nf

```diff
@@ -3,6 +3,32 @@
 // Specify DSL2
 nextflow.enable.dsl=2
 
+/*------------------------------------------------------------------------------------*/
+/* Processes
+--------------------------------------------------------------------------------------*/
+
+process count_lines {
+    label "low_cores"
+    label "low_mem"
+    label "regular_queue"
+
+    tag "${sample_id}"
+
+    container 'ubuntu:16.04'
+
+    input:
+        tuple val(sample_id), path(input_file)
+
+    output:
+        tuple val(sample_id), stdout, emit: line_count
```

Comment:
whats the stdout line?

Reply:
Outputs the stdout of the command to a Nextflow variable: https://www.nextflow.io/docs/latest/process.html#output-stdout-special-file

Reply:
ah thats cool didnt realise you could do that